.gitignore

```diff
@@ -0,0 +1,8 @@
+.idea
+.DS_Store
+.sass-cache
+
+## Directories
+log/
+dist/
+node_modules/
```

Comment:
Not good enough maybe [link](https://gist.github.com/andreasonny83/b24e38b7772a3ea362d8e8d238d5a7bc?fbclid=IwAR2cypSrzIIJc4G7ZNqSrUcd9VUQ-f-ZUlH1TUOHt6Ur7u8FDHsqf0IGMiE)